scripts/upload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+cd ..
```

Comment:
This is dangerous -- scripts should not be required to be run in a certain directory